src/uv_mbed.c

```diff
@@ -10,10 +10,29 @@
 #include <assert.h>
 #include "uv_mbed/uv_mbed.h"
 #include "bio.h"
+#include "uv-common.h"
+
+#if _WIN32
+// this function is declared INLINE in an mbed .h file. As such we have had to 
```

Comment:
comment wrong? it's in libuv header file

Reply:
changed to libuv - thx